settings/defaults.php

```diff
@@ -1,6 +1,6 @@
 <?php
-@define('CONST_BasePath', '@CMAKE_SOURCE_DIR@');
-@define('CONST_InstallPath', '@CMAKE_BINARY_DIR@');
+@define('CONST_BasePath', '@WWW_APP@');
+@define('CONST_InstallPath', '@WWW_PUBLIC@');
```

Comment:
I'm not sure, what you want to achieve here. BasePath and InstallPath point to the directory where to find data, php scripts and stuff. So you can't set them to some global path without installing all these files. Also note that they are currently set up so that Nominatim can be executed directly from the build directory. This is something that needs to remain possible for the development process. So we'd really need a separate 'make install' here, which creates its own settings suitable for the installed version.